src/main/java/org/broadinstitute/hellbender/tools/spark/sv/utils/FileUtils.java

```diff
@@ -34,7 +34,10 @@ public static void writeSAMFile(final Iterator<SAMRecord> alignments, final SAMF
         }
     }
 
-    public static boolean createDirInBucketToWriteTo(final String pathString) {
+    /**
+     * Creates a directory, in local FS, HDFS, or Google buckets to write to.
+     */
+    public static boolean createDirToWriteTo(final String pathString) {
```

Comment:
In this method, I think that you could use IOUtils.getPath() you could combine this clause with the third clause.